tests/normalize-data-types.test.js

```diff
@@ -0,0 +1,41 @@
+import { normalize } from '../src/util/dataTypes'
+
+let table = [
+  ['foo', 'foo'],
+  ['foo-bar', 'foo-bar'],
+  ['16/9', '16 / 9'],
+
+  // '_'s are converted to spaces except when escaped
+  ['foo_bar', 'foo bar'],
+  ['foo__bar', 'foo  bar'],
+  ['foo\\_bar', 'foo_bar'],
+
+  // Urls are preserved as-is
+  [
+    'url("https://example.com/abc+def/some-path/2022-01-01-abc/some_underscoered_path")',
+    'url("https://example.com/abc+def/some-path/2022-01-01-abc/some_underscoered_path")',
+  ],
+
+  // var(…) is preserved as is
+  ['var(--foo)', 'var(--foo)'],
+  ['var(--headings-h1-size)', 'var(--headings-h1-size)'],
+
+  // calc(…) get's spaces around operators
+  ['calc(1+2)', 'calc(1 + 2)'],
+  ['calc(100%+1rem)', 'calc(100% + 1rem)'],
```

Comment:
I would maybe add 1 more test with nested `calc` just to verify that that works as well `calc(1+calc(100%-20px))` for example should result in `calc(1 + calc(100% - 20px))`

Reply:
Added and it works as expected 🎉